.github/workflows/heavy-integration.yml

```diff
@@ -10,94 +10,34 @@ env:
-                  --locked \
-                  --archive-file ${{ matrix.archive.file }} \
-                  ${{ matrix.archive.flags }}
-            - name: Upload archive
-              uses: actions/upload-artifact@v3
-              with:
-                  name: ${{ matrix.archive.name }}
-                  path: ${{ matrix.archive.file }}
-
-    install-svm-solc:
-        name: install svm and solidity / ${{ matrix.job.name }}
-        runs-on: ubuntu-latest
-        timeout-minutes: 30
-        steps:
-            - name: Install svm
-              run: cargo install svm-rs
-            - name: Install Solidity 0.8.19
-              run: svm install 0.8.19
-            - name: Install Solidity 0.8.20
-              run: svm install 0.8.20
-            - name: Use Solidity 0.8.19
-              run: svm use 0.8.19
-
     heavy-integration:
-        name: heavy (long-running) integration tests / ${{ matrix.job.name }}
+        name: heavy (long-running) integration tests
         runs-on: ubuntu-latest
-        needs: build-tests
-        strategy:
-            matrix:
-                job:
-                    - name: Long-running integration tests
-                      filter: "!test(~live) & test(heavy)"
         env:
             ETH_RPC_URL: https://eth-mainnet.alchemyapi.io/v2/C3JEvfW6VgtqZQa-Qp1E-2srEiIc02sD
         steps:
             - uses: actions/checkout@v3
-            - uses: taiki-e/install-action@nextest
             - uses: dtolnay/rust-toolchain@stable
-            - name: Download archives
-              uses: actions/download-artifact@v3
-              with:
-                  name: heavy-integration-tests
-
+            - uses: taiki-e/install-action@nextest
+            - uses: Swatinem/rust-cache@v2
             - name: Forge RPC cache
               uses: actions/cache@v3
-              if: matrix.job.name != 'non-forking'
               with:
-                  path: "$HOME/.foundry/cache"
+                  path: |
+                      ~/.foundry/cache
+                      ~/.config/.foundry/cache
                   key: rpc-cache-${{ hashFiles('crates/forge/tests/rpc-cache-keyfile') }}
-
-            - name: Setup git config
-              run: |
-                  git config --global user.name "GitHub Actions Bot"
-                  git config --global user.email "<>"
-
-            - name: Force use of HTTPS for submodules
-              run: git config --global url."https://github.com/".insteadOf "git@github.com:"
-
-            - name: cargo nextest
+            - name: test
               run: |
-                  # see https://github.com/foundry-rs/foundry/pull/3959
-                  export LD_LIBRARY_PATH="$(rustc --print sysroot)/lib"
-                  cargo nextest run --retries 3 --archive-file heavy-integration.tar.zst -E '${{ matrix.job.filter }}'
+                  cargo nextest run -r -p forge --test cli --features heavy-integration-tests --retries 1 -E 'test(~heavy_integration)'
```

Comment:
Hmm is the matcher here correct? I think this will try to match the whole string, so you'd have to prefix the maple heavy integration test with `heavy_integration`

Reply:
`test([~]<string>)` -> `test_name.contains("<string>")` where test_name is the full path

You can check with:

```sh
$ cargo nextest list -p forge --test cli --features heavy-integration-tests -E 'test(~heavy_integration)'
    Finished test [unoptimized] target(s) in 20.85s
forge::cli:
    heavy_integration::heavy_maple
```